### PR TITLE
Deprecate module attributes XXYY, YXXY, CXXYY, CYXXY

### PR DESCRIPTION
**Pull request by kevinsung**
Achieves this by wrapping the modules in a class that defines `__getattr__`. See https://stackoverflow.com/questions/922550/how-to-mark-a-global-as-deprecated-in-python. The diff for the first commit may be easier to read; the second one is just formatting.

### CODE REVIEW

File: openfermioncirq/gates/common_gates.py [hunk 1]
```diff
@@ -40,23 +39,22 @@ class FSwapPowGate(cirq.EigenGate,
...
-            (1, np.array([[0,  0,    0,   0],
-                          [0,  0.5, -0.5, 0],
-                          [0, -0.5,  0.5, 0],
-                          [0,  0,    0,   1]])),
+            (0,
+             np.array([[1, 0, 0, 0], [0, 0.5, 0.5, 0], [0, 0.5, 0.5, 0],
+                       [0, 0, 0, 0]])),
+            (1,
+             np.array([[0, 0, 0, 0], [0, 0.5, -0.5, 0], [0, -0.5, 0.5, 0],
+                       [0, 0, 0, 1]])),
```
Comment (viathor) [inline openfermioncirq/gates/common_gates.py:53]: Optional nit: Previous formatting is clearly more readable. Consider displaying yapf for these four lines. [Example](https://github.com/quantumlib/Cirq/blob/9065d4e86720bcd2d68eff808663ee4f2106eeee/cirq/ops/swap_gates_test.py#L74).

Comment (kevinsung) [reply openfermioncirq/gates/common_gates.py:58]: Done.

**Comment by viathor** [comment]
Thanks!

**Comment by kevinsung** [comment]
Not sure why Travis is suddenly asking me to lint every file but so be it...

**Pull request merged**